### DISCUSSION TITLE
log changes also in user actions log of Joomla ?

### DISCUSSION THREAD

**Discussion by Rolli1962** [akeeba/panopticon#387] [General]
Would it be possible to log changes also in the User Actions Log of Joomla ?

In case of one of the sites suddenly makes some trouble. I think the first way would be to go in the back-end and check what has be changed in the last time via the User Actions Log. Currently an update via panopticon is not logged there. And it would take a while better to see if there was a change via panopticon.

Maybe in addition there is a responsible Admin for only one site, try to find out and has no access-rights to panopticon. So even such a person has a chance to see what has be changed via panopticon.

Hope those (additional) logging makes sense?

**Comment by nikosdion** [comment]
No, the entire point of having an API is that it is a non-interactive method of doing stuff on your site, therefore it is not logged into the user actions log.

**Reply by Rolli1962** [reply]
Thanks for answer. Hoped it would be an easy way to activate. And OK - if such a feature it is not provided by Joomla in general it is nearly impossible.

**Reply by nikosdion** [reply]
It's not so much that it's not provided, it's that it would not work the way you would think it should work. The authentication is performed in the level of the API application. This is always a Super User; remember, you need a Super User to create an API key to the Joomla! API application. Therefore, Joomla! always “sees” that Super User whose API key we provided as the logged in user. As a result, you'd only get a log entry saying that it's this Super User making changes.

Now, keep in mind that the Super User privilege is “God Mode“. If you do not absolutely and perfectly trust a person, don't give them Super User privileges on your site. If you do not absolutely and perfectly trust a person, don't give them Admin privileges in Panopticon.
